nginx/init.sls

```diff
@@ -19,3 +19,20 @@ nginx:
   file.symlink:
     - target: /etc/nginx/sites-available/default
 
+certbot:
+  pkgrepo.managed:
+    - ppa: certbot/certbot
+  pkg.installed:
+    - pkgs:
+      - certbot
+      - python-certbot-nginx
+
+certbot renew:
+  cron.present:
+    - identifier: build-cert-renew
+    - user: root
+    - minute: 0
+    - hour: 0
+    - daymonth: 1
```

Comment:
Instead of running once a month, it would be nice to run once a week (ish) so we can catch problems faster. Also, we should figure out a story for getting notifications of failed jobs.